### PR TITLE
[Spike] Decorate RPC methods

### PR DESCRIPTION
**Pull request by aidan-starke**
## Summary

 - [TRNS-148](https://centralitydev.atlassian.net/jira/software/projects/TRNS/boards/149?selectedIssue=TRNS-148)
 - Add RPC method definitions for rootnet

## Checklist

- [x] Add description
- [x] Tag related issue(s)

### CODE REVIEW

**Review by ken-centrality — COMMENTED** [review]
Got this error

```
Uncaught error. Something went wrong with the query and rendering of this component. Please supply all the details below when logging an issue, it may help in tracing the cause.

Cannot read properties of null (reading 'toHuman')
```

When trying to fetch EventProof ID `940`

**Comment by aidan-starke** [comment]
@ken-centrality sorted now

**Review by KarishmaBothara — APPROVED** [review]

File: packages/apps-config/src/api/spec/rootnet.ts [hunk 1]
```diff
@@ -18,15 +19,172 @@ const definitions: OverrideBundleDefinition = {
...
+        description: 'Get XRPL event proof for event Id',
+        params: [
+          {
+            name: 'eventId',
+            type: 'EventProofId'
+          }
+        ],
+        type: 'Option<XrplEventProofResponse>'
+      },
+      subscribeEventProofs: {
```
Comment (KarishmaBothara) [inline packages/apps-config/src/api/spec/rootnet.ts:148]: I think we can remove the subscribe/unsubscribe event proof... as we don't generally add subsribe/unsubscribe in rpc here..
Screenshot 2023-03-13 at 5 32 16 PM
Screenshot 2023-03-13 at 5 33 27 PM

Comment (aidan-starke) [reply packages/apps-config/src/api/spec/rootnet.ts:148]: Yeah good call, @ken-centrality I'm thinking we take these out for now, then in the npm package we can add a flag `includeSubscriptions` to optionally include these, what do you think?

Comment (ken-centrality) [reply packages/apps-config/src/api/spec/rootnet.ts:148]: yup that's fine, maybe just comment it out

**Review by ken-centrality — DISMISSED** [review]
LGTM, merge once Karishma is happy

**Pull request merged**